packages/badge/src/component.tsx

```diff
@@ -0,0 +1,50 @@
+import { classNames } from '@chbphone55/classnames';
+//import { badge as ccBadge } from '@warp-ds/css/component-classes';
+const ccBadge = {
+    base: 'py-4 px-8 border-0 rounded-4 text-xs inline-flex',
+    neutral: 'i-bg-$color-badge-neutral-background i-text-$color-badge-neutral-text',
+    info: 'i-bg-$color-badge-info-background i-text-$color-badge-info-text',
+    positive: 'i-bg-$color-badge-positive-background i-text-$color-badge-positive-text',
+    warning: 'i-bg-$color-badge-warning-background i-text-$color-badge-warning-text',
+    negative: 'i-bg-$color-badge-negative-background i-text-$color-badge-negative-text',
+    disabled: 'i-bg-$color-badge-disabled-background i-text-$color-badge-disabled-text',
+    price: 'i-bg-$color-badge-price-background i-text-$color-badge-price-text',
+    notification: 'i-bg-$color-badge-notification-background i-text-$color-badge-notification-text',
+    positionBase: 'absolute backdrop-blur',
+    positionTL: 'rounded-tl-0 rounded-tr-0 rounded-bl-0 top-0 left-0',
+    positionTR: 'rounded-tl-0 rounded-tr-0 rounded-br-0 top-0 right-0',
+    positionBR: 'rounded-tr-0 rounded-br-0 rounded-bl-0 bottom-0 right-0',
+    positionBL: 'rounded-tl-0 rounded-br-0 rounded-bl-0 bottom-0 left-0',
+};
+import React from 'react';
+import { BadgeProps } from './props';
+
+export function Badge(props: BadgeProps) {
+  const {
+    children,
+    as = 'div',
+    type = 'neutral',
+    position,
+    ...rest
+  } = props;
+
+  return React.createElement(
+    as,
+    {
+      ...(rest as Omit<BadgeProps, 'children'> as {}),
+      className: classNames(
+        ccBadge.base,
+        ccBadge[type],
+        {
+            [ccBadge.positionBase]: !!position,
+            [ccBadge.positionTL]: position === 'tl',
+            [ccBadge.positionTR]: position === 'tr',
+            [ccBadge.positionBR]: position === 'br',
+            [ccBadge.positionBL]: position === 'bl',
+        },
+        props.className,
+      ),
+    },
```

Comment:
Not sure about this one but I can see Shoelace web component library sets `role="status"` on their badge component ([see here](https://github.com/shoelace-style/shoelace/blob/next/src/components/badge/badge.component.ts#L44C9-L44C22)). Here's some info about role=status from WCAG docs: https://www.w3.org/WAI/WCAG22/Techniques/aria/ARIA22
It's not easy for me to see if that applies to our Badge but maybe the someone from the accessibility group could help us here. Wdyt?

Reply:
I guess that is totally up to the consumers what they use it for? 
In Vue they can easily just do `<w-badge role="status">A badge with role</w-badge>`... but it is not as easy in React or Elements? Do we need to add props for this? 🤷 Personally I would think that it should be up to the consumers to add any attribute they want on any component's wrapping element?

Reply:
Badge is also used as a price tag... which I guess wouldn't make sense to put `aria-label="status"` to?

Reply:
Ideally we should prob not use this component for pricetag.  And ideally it should probably be role="status" if that is always the case.  Im a bit conflicted about setting Aria roles on stuff as defaults when its not always correct, in fact that creates more issues than it solves in most cases.